fanout/assemble_fanout_group.js

```diff
@@ -0,0 +1,206 @@
+const EventEmitter = require('events');
+
+const asyncRetry = require('async/retry');
+const {broadcastChainTransaction} = require('ln-service');
+const {combinePsbts} = require('psbt');
+const {decodePsbt} = require('psbt');
+const {extractTransaction} = require('psbt');
+const {finalizePsbt} = require('psbt');
+const {Transaction} = require('bitcoinjs-lib');
+
+const coordinateFanout = require('./coordinate_fanout');
+const {peerWithPartners} = require('./../groups/p2p');
+const proposeFanout = require('./propose_fanout');
+const {signAndFundGroupChannel} = require('../groups/funding');
+
+const {fromHex} = Transaction;
+const interval = 500;
+const minGroupCount = 3;
+const times = 2 * 60 * 10;
+
+/** Assemble a fanout group
+
+  {
+    capacity: <Output Capacity Tokens Number>
+    count: <Channel Members Count Number>
+    ecp: <ECPair Library Object>
+    identity: <Coordinator Identity Public Key Hex String>
+    [inputs]: [<Utxo Outpoint String>]
```

Comment:
This should be required, to say which inputs you want to be paid into the fanout